javascript/apps/taiga/src/app/shared/comments/components/deleted-comment/deleted-comment.component.css

```diff
@@ -0,0 +1,16 @@
+/*
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+*/
+
+.deleted-message {
+  background-color: #f9f9fb;
+  color: #63738f;
+  font-style: italic;
+  margin-block-end: var(--spacing-16);
+  padding-block: 8px;
+  padding-inline: 16px;
```

Comment:
Use variables for sizing and colors

Reply:
Not sure if applies here, but for text styles there is a list of mixins that can be applied at `typography.css`